src/screens/AccountRecover.js

```diff
@@ -53,124 +59,121 @@ class AccountRecoverView extends React.Component {
   constructor(...args) {
     super(...args);
 
-    this.state = { seed: ''}
+    this.state = { seed: '' };
   }
 
-  addressGeneration = (seed) => {
+  addressGeneration = seed => {
     const { accounts } = this.props;
-    
+
     brainWalletAddress(seed)
-    .then(({ address, bip39 }) => accounts.updateNew({address, seed, validBip39Seed: bip39}))
-    .catch(console.error);
-  }
+      .then(({ address, bip39 }) =>
+        accounts.updateNew({ address, seed, validBip39Seed: bip39 })
+      )
+      .catch(console.error);
+  };
 
-  debouncedAddressGeneration = debounce(this.addressGeneration, 200)
-  
-  componentWillUnmount = function () {
+  debouncedAddressGeneration = debounce(this.addressGeneration, 200);
+
+  componentWillUnmount = function() {
     // called when the user goes back, or finishes the whole recovery process
-    this.props.accounts.updateNew({seed : ''});
-  }
+    this.props.accounts.updateNew({ seed: '' });
+  };
 
   render() {
     const { accounts } = this.props;
```

Comment:
(nit), it's not from your PR, but as I see it..
```suggestion
    const { accounts, navigation } = this.props;
```